deployment/gcp/crawl.tmpl.yaml

```diff
@@ -63,6 +63,8 @@ spec:
           value: 'DEBUG'
         - name: LOG_LEVEL_SENTRY_EVENT
           value: 'ERROR'
+        - name: MAX_JOB_RETRIES
+          value: '3'
```

Comment:
Note: [Scrapy has a default of 2](https://docs.scrapy.org/en/latest/topics/downloader-middleware.html#retry-times)

Reply:
Fixed.